sw/airborne/subsystems/gps/librtcm3/rtcm3.h

```diff
@@ -363,9 +365,11 @@ s8 rtcm3_process(msg_state_t *s, unsigned char buff)
         // Check what message type it is
         switch (RTCMgetbitu(s->msg_buff, 24 + 0, 12)) {
           case 1005: s->msg_type = RTCM3_MSG_1005; break;
+          case 4072: s->msg_type = RTCM3_MSG_4072; break;
           case 1077: s->msg_type = RTCM3_MSG_1077; break;
           case 1087: s->msg_type = RTCM3_MSG_1087; break;
-          default  : printf("Unknown message type\n"); return RTCM_OK_CALLBACK_UNDEFINED;
+          case 1230: s->msg_type = RTCM3_MSG_1230; break;
+          default  : printf("Unknown message type %u\n", RTCMgetbitu(s->msg_buff, 24 + 0, 12)); return RTCM_OK_CALLBACK_UNDEFINED;
```

Comment:
This `printf`should be protected with `DEBUG_PRINT_PACKAGE` like the others. It was not the case, but better make it right now and prevent a print on system that might not support it.

Reply:
Yes, I agree. I added the changes.

Reply:
Sorry to be annoying, but only the print should be protected, the default case with return code is still needed. It should be something like:
```
  default:
#ifdef DEBUG_PRINT_PACKAGE
    printf("Unknown message type %u\n", RTCMgetbitu(s->msg_buff, 24 + 0, 12));
#endif
    return RTCM_OK_CALLBACK_UNDEFINED;
```
As a side note, when creating pull-requests you can allow maintainers to make direct code suggestions (instead of simple comment) to make little changes like this one easier: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
Oh sure, I overlooked that yesterday. I just added the changes (and edits from maintainers are enabled as well now). 
And thank you for the info, still have to get used to git.